src/test/resources/pages/studentFeedbackSubmitPageSuccessRank.html

```diff
@@ -5,6 +5,12 @@
     Submit Feedback
   </h1>
   <br>
+  <div id="student-motd-wrapper">
+    <input id="motd-url" type="hidden" value="${studentmotd.url}">
+    <script defer="" src="/js/studentMotd.js" type="text/javascript">
+    </script>
+    ${studentmotd.container}
+  </div>
```

Comment:
It comes from `teammates.test.cases.browsertests.FeedbackRankQuestionUITest` > `testStudentSubmitAndResultsPages()`. Yes it does look like undesired code. Nice catch! :+1: But I'm not sure how to deal with it. All my tests are passing without godmode now. :sweat_smile: